general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -192,17 +191,41 @@ def get_job(self, job_id: str) -> dict[str, str]:
         """
         return self.fetch_jobs([job_id])[job_id]
 
+    def cancel_jobs(
+        self,
+        job_ids: Sequence[str],
+        **kwargs: object,
+    ) -> list[str]:
+        """Cancel jobs associated with given job ids.
+
+        Args:
+            job_ids: The UUIDs of the jobs (returned when the jobs were created).
+            kwargs:  Extra options needed to fetch jobs.
+
+        Returns:
+            A list of the job ids of the jobs that successfully cancelled.
+
+        Raises:
+            SuperstaqServerException: For other API call failures.
+        """
+        json_dict: dict[str, Any] = {
```

Comment:
```suggestion
        json_dict: dict[str, list[str]] = {
```

Reply:
ah my bad. what about this?
```suggestion
        json_dict: dict[str, str | Sequence[str]] = {
```